packages/styles/stories/introduction.stories.mdx

```diff
@@ -0,0 +1,52 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Styles/About the Style injection system" />
+
+# About the Salt Theme
+
+## Component style injection
+
+In order to support uses of Salt that utilize multiple external windows such as desktop applications. We offer two packages that contain hooks to facilitate this: `@salt-ds/window` and `@salt-ds/styles`.
```

Comment:
`desktop applications` is not a well understood concept among web devs, could illustrate more on what it is and what exact problem to be addressed.